simd/powerpc/jsimd.c

```diff
@@ -122,6 +125,9 @@ init_simd(void)
   int mib[2] = { CTL_MACHDEP, CPU_ALTIVEC };
   int altivec;
   size_t len = sizeof(altivec);
+#elif defined(__FreeBSD__)
+  extern u_long cpu_features;
+  int altivec;
```

Comment:
I don't think you need `int altivec;` here.

Reply:
You're right, thanks.